index.html

```diff
@@ -59,13 +60,12 @@
 
   </div>
 
-  <script src="//cdn.pypyjs.org/pypy.js/0.3.1/lib/Promise.min.js" type="text/javascript"></script>
-  <script src="//cdn.pypyjs.org/pypy.js/0.3.1/lib/FunctionPromise.js" type="text/javascript"></script>
-  <script src="//cdn.pypyjs.org/pypy.js/0.3.1/lib/pypy.js" type="text/javascript"></script>
+  <script src="pypyjs-release/lib/Promise.min.js" type="text/javascript" charset="utf-8"></script>
+  <script src="pypyjs-release/lib/FunctionPromise.js" type="text/javascript" charset="utf-8"></script>
+  <script src="pypyjs-release/lib/pypy.js" type="text/javascript" charset="utf-8"></script>
```

Comment:
* less CDN transfer volumen
* faster unittests
* it's possible to use a other branch of the "pypyjs-release" repro